tests/test_mtdome_csc.py

```diff
@@ -536,15 +539,8 @@ async def test_do_park(self) -> None:
                 self.csc.mock_ctrl.current_tai + START_MOTORS_ADD_DURATION + 0.1
             )
 
-            # Now also check the azMotion event.
-            await self.csc.statusAMCS()
-            amcs_status = self.csc.lower_level_status[mtdome.LlcName.AMCS.value]
-            assert amcs_status["status"]["status"] == MotionState.PARKED.name
-            await self.assert_next_sample(
-                topic=self.remote.evt_azMotion,
-                state=MotionState.PARKED,
-                inPosition=True,
-            )
+            # No new azMotion event gets emitted since the PARKED event already
```

Comment:
I think the code at line 537 is no longer relevant. you change mock_ctrl.current_tai but don't do anything to test for the effect of that.

If you do want to test for state I suppose you could still check the state of the CSC. Checking events is trickier and may well not be worth the hassle. I sometimes read until I get the value I'm looking for (which times out if the value never comes). But it can be hard to be sure that the match you find is for the most recent message.

Reply:
Good catch. I removed the lines.